packages/botonic-react/src/webchat/styled-scrollbar.jsx

```diff
@@ -0,0 +1,48 @@
+import styled from 'styled-components'
+import SimpleBar from 'simplebar-react'
+import 'simplebar/dist/simplebar.css'
+
+export const StyledScrollbar = styled(SimpleBar)`
+  & .simplebar-scrollbar::before {
+    background-color: ${({ scrollbar }) =>
+      scrollbar && scrollbar.thumb && scrollbar.thumb.color
```

Comment:
I wish we already had this in js :-) https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
this would be awesome. what about this [babel-plugin](https://babeljs.io/docs/en/babel-plugin-proposal-optional-chaining)?